install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -372,13 +372,13 @@ spec:
         - sh
         - -c
         - |
-          until test -s {{ .Values.nodeinit.bootstrapFile | quote }}; do
+          until test -s {{ (print "/tmp/cilium-bootstrap.d/" (.Values.nodeinit.bootstrapFile | base)) | quote }}; do
             echo "Waiting on node-init to run...";
             sleep 1;
           done
         volumeMounts:
-        - name: cilium-bootstrap-file
-          mountPath: {{ .Values.nodeinit.bootstrapFile }}
+        - name: cilium-bootstrap-file-dir
+          mountPath: "/tmp/cilium-bootstrap.d"
```

Comment:
We mount the bootstrap dir in a known location, so that even if users ask to store this in `/`, we won't override any known path in the container.